packages/conform-zod/parse.ts

```diff
@@ -105,18 +106,23 @@ export function refine(
 		/**
 		 * The message displayed when the validation fails.
 		 */
-		message: string;
+		message?: string;
 		/**
 		 * The path set to the zod issue.
 		 */
 		path?: IssueData['path'];
+		/**
+		 * Custom parameters
+		 */
+		params?: ZodCustomIssue['params'];
 	},
 ): void | Promise<void> {
 	if (typeof options.when !== 'undefined' && !options.when) {
 		ctx.addIssue({
 			code: ZodIssueCode.custom,
 			message: VALIDATION_SKIPPED,
 			path: options.path,
+			params: options.params,
```

Comment:
Both `VALIDATION_SKIPPED` and `VALIDATION_UNDEFINED` are custom messages that Conform used internally and they should never be translated. Can we skip the `params` for these two issues? I assume zod-i18n will just ignore them and leave them as is if there is no `params` set?

Reply:
The zod-i18n translates it only if you pass it under 18n property, like 
```ts
params: {
  i18n: "translationKey"
}
```

Do you think it's still gonna be an issue?

Reply:
@edmundhung A week gone by and it finally clicked what you meant 🥲 Yeah, i removed the params for the invalid cases, now it's passed only at the end